src/multi_physics/CMakeLists.txt

```
@@ -1,13 +1,23 @@
-cmake_minimum_required (VERSION 3.5)
+cmake_minimum_required(VERSION 3.12)
 
-set(CMAKE_CXX_EXTENSIONS OFF)
+project(
+  PXRMP
+  DESCRIPTION
+    "A header only c++ library (part of the PICSAR library) providing QED modules for Particle-In-Cell codes."
+  HOMEPAGE_URL "https://github.com/ECP-WarpX/picsar"
+  LANGUAGES CXX)
```

Let's avoid this line, since practically we do also depend on CUDA when using the interface libraries herein:
```suggestion
  HOMEPAGE_URL "https://github.com/ECP-WarpX/picsar")
```